### PR TITLE
Scripting to create a GKE Cluster and install operator

### PR DESCRIPTION
**Pull request by chrislovecnm**
This PR includes some shell scripts that allow for
- the creation of a GKE cluster
- build and push the operator container into gcr
- install the operator
- install a basic example
- other scripts that allow you to cleanup

### CODE REVIEW

**Comment by chrislovecnm** [comment]
cc @chrisseto @johnrk

**Comment by chrislovecnm** [comment]
@johnrk this is blocked by lack of GCP account

**Comment by chrislovecnm** [comment]
cc @chrisseto PTAL

File: config/examples/3-nodes-insecure.yaml [hunk 1]
```diff
@@ -11,7 +11,8 @@ spec:
         resources:
           requests:
             storage: 60Gi
-        storageClassName: crdb-io1
+        # this needs to be precreated and we are not checking this
```
Comment (chrisseto) [inline config/examples/3-nodes-insecure.yaml:14]: Can this block be removed?

File: hack/apply-crdb-example.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# "---------------------------------------------------------"
+# "-                                                       -"
+# "-  deploy the operator 				   -"
```
[the remark this answers is not in the document]
Comment (chrislovecnm) [reply hack/apply-crdb-example.sh]: Checking to see if it is fix.  I think it is a tab vs spaces problem.

File: hack/apply-operator.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# "---------------------------------------------------------"
+# "-                                                       -"
+# "-  deploy the operator 				   -"
```
[the remark this answers is not in the document]
Comment (chrislovecnm) [reply hack/apply-operator.sh]: Checking to see if it is fix. I think it is a tab vs spaces problem.

**Pull request merged**